modules/openapi-generator/src/main/resources/cpp-qt-client/api-header.mustache

```diff
@@ -93,7 +93,7 @@ signals:
 {{#operations}}{{#operation}}
     void {{nickname}}SignalE({{#returnType}}{{{.}}} summary, {{/returnType}}QNetworkReply::NetworkError error_type, QString error_str);{{/operation}}{{/operations}}
```

Comment:
I realize I should have change this signal signature too. Waiting for your thoughts about my last comment.